.github/workflows/bench-pr.yml

```diff
@@ -33,9 +33,11 @@ jobs:
           version: v1.0.0
 
       - name: Run benchmark
+        shell: bash
```

Comment:
if you don't set this to bash you don't get `-o pipefail` which was the reason things succeeded